app/mixins/update-selection-all-features.js

```diff
@@ -10,11 +10,12 @@ import { inject as service } from '@ember/service';
 
 export default Mixin.create({
   mainMap: service(),
-  afterModel({ taskInstance }, transition) {
-    if (transition.queryParams.search === 'true') {
-      this.waitToFitBounds.perform(taskInstance);
-    }
-  },
```

Comment:
This all seems a little heavy-handed, no? Why don't you just add a defensive check on line 13? Then we don't have to comment out your enhancement :)

```
if (transition.queryParams && transition.queryParams.search === 'true') {
  // stuff
}
```

Reply:
Oh yes much better--I'll make that change

Reply:
Done!